open-metadata-implementation/framework-services/ocf-metadata-management/ocf-metadata-client/src/main/java/org/odpi/openmetadata/frameworkservices/ocf/metadatamanagement/client/ConnectedAssetClientBase.java

```diff
@@ -483,6 +483,41 @@ protected String  getAssetForConnection(OCFRESTClient  restClient,
     }
 
 
+    /**
+     * Returns the anchor asset of the supplied entity.
+     *
+     * @param userId the userId of the requesting user.
+     * @param entityGUID unique identifier for the entity.
+     *
+     * @return anchor asset of the entity.
+     *
+     * @throws InvalidParameterException one of the parameters is null or invalid.
+     * @throws PropertyServerException there is a problem retrieving information from the property server.
+     * @throws UserNotAuthorizedException the requesting user is not authorized to issue this request.
+     */
+    public Asset getAnchorAssetFromGUID(String userId,
+                                        String entityGUID) throws InvalidParameterException,
```

Comment:
I suggest changing entityGUID to guid. As this API does not know about entities as a concept.